tests/unit/amo/components/TestSearchForm.js

```diff
@@ -149,4 +150,45 @@ describe(__filename, () => {
 
     sinon.assert.notCalled(fakeHistory.push);
   });
+
+  it('renders a Helmet', () => {
+    const root = render();
+
+    const helmet = root.find(Helmet);
+    expect(helmet).toHaveLength(1);
+
+    const link = helmet.find('link');
+    expect(link).toHaveLength(1);
+    expect(link).toHaveProp('rel', 'search');
+    expect(link).toHaveProp('type', 'application/opensearchdescription+xml');
+    expect(link).toHaveProp('href');
+    expect(link).toHaveProp('title');
+  });
+
+  it('renders opensearch link when client App is Android and lang is en-CA', () => {
+    const { store } = dispatchClientMetadata({
+      clientApp: CLIENT_APP_ANDROID,
+      lang: 'en-CA',
+    });
+    const root = render({ store });
+    const link = root.find('link');
+
+    expect(link).toHaveProp(
+      'href',
+      `/en-CA/${CLIENT_APP_ANDROID}/opensearch.xml`,
+    );
+    expect(link).toHaveProp('title', 'Firefox Add-ons (en-CA)');
+  });
+
+  it('renders opensearch link when client App is Firefox and lang is fr', () => {
```

Comment:
I added two tests for different `clientapp` and `lang`, but not sure if I should keep both.

Reply:
that's perfect actually, and it makes me think that we should have a better title for Android. I am going to push a commit on top of your commits :)